packages/cli/src/commands/generate/index.ts

```diff
@@ -15,14 +17,44 @@ export const generateSchema = async ({
     defaultSchema,
     schemas = [],
     tables = [],
+    schemaDefOverrides,
 }: GenerateSchemaOptions) => {
-    const result = await generate({
-        defaultSchema,
-        connectionString,
-        includeSchemas: schemas,
-        includeTables: tables,
-        outDir: out,
-    });
+    if (schemaDefOverrides === undefined) {
+        try {
+            return await generate({
+                defaultSchema,
+                connectionString,
+                includeSchemas: schemas,
+                includeTables: tables,
+                outDir: path.resolve(path.join(process.cwd(), out)),
+            });
+        } catch (error) {
```

Comment:
Don't catch an error just to log it. This isn't really doing anything. Just let the error bubble up.

Reply:
Same for the other try/catch blocks in this file.